ballerina/Package.md

```diff
@@ -12,8 +12,9 @@ The `ballerinax/salesforce` is a [Ballerina](https://ballerina.io/) connector fo
 |                     | Version            |
 |---------------------|--------------------|
 | Ballerina Language  | Swan Lake 2201.8.0 |
-| Salesforce REST API | v48.0              |
-| Salesforce Bulk API | v1                 |
+| Salesforce REST API | v59.0              |
```

Comment:
As we don't use Package.md in central anymore, Package.md should duplicate the exact Module.md content

Reply:
updated in [5452761](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/319/commits/54527610e82929dda4ebc8f90016a66c39eb1501)